new-packages.json

```
@@ -143,5 +143,6 @@
   "purescript-google-apps": "https://github.com/BBVA/purescript-google-apps.git",
   "purescript-peregrine": "https://github.com/maxdeviant/peregrine.git",
   "purescript-datetime-parsing": "https://github.com/flounders/purescript-datetime-parsing.git",
-  "purescript-run-external-state": "https://github.com/Mateiadrielrafael/purescript-run-external-state.git"
+  "purescript-run-external-state": "https://github.com/Mateiadrielrafael/purescript-run-external-state.git",
+  "purescript-ask": "https://github.com/Mateiadrielrafael/purescript-ask.git",
```

I think the tests don't allow trailing commas -- sorry it's so pedantic 😬 
```suggestion
  "purescript-ask": "https://github.com/Mateiadrielrafael/purescript-ask.git"
```

---

Oh, thanks